packages/design-tokens-schema/src/basis-tokens.ts

```diff
@@ -316,28 +306,31 @@ export const StrictThemeSchema = ThemeSchema.transform(addContrastExtensions)
       for (const { color: background, expectedRatio } of comparisons) {
         const compareColor = getActualValue<ColorValue>(background);
         const contrast = compareContrast(baseColor, compareColor);
-        const resolvedFrom = background.$extensions?.[EXTENSION_RESOLVED_FROM];
-        const colorRefName = background.$extensions?.[EXTENSION_RESOLVED_AS]
-          ? `"${resolvedFrom}" -> "${background.$value}"`
-          : `"${resolvedFrom}"`;
+        const tokenAPath = path.join('.');
+        const tokenBPathRaw = background.$extensions?.[EXTENSION_RESOLVED_FROM] as string | undefined;
+        const tokenBPath = tokenBPathRaw?.replace(/^\{|\}$/g, '');
+
         if (contrast < expectedRatio) {
           ctx.addIssue({
+            actual: Number.parseFloat(contrast.toFixed(2)),
             code: 'too_small',
             ERROR_CODE: ERROR_CODES.INSUFFICIENT_CONTRAST,
-            input: contrast,
-            message: `Not enough contrast between "{${path.join('.')}}" (${stringifyColor(baseColor)}) and ${colorRefName} (${stringifyColor(compareColor)}). Calculated contrast: ${contrast.toFixed(2)}, need ${expectedRatio}`,
+            message: 'Insufficient contrast',
             minimum: expectedRatio,
             origin: 'number',
             path: [...path, '$value'],
+            tokens: [tokenAPath, tokenBPath].filter(Boolean),
           });
+
           ctx.addIssue({
+            actual: Number.parseFloat(contrast.toFixed(2)),
```

Comment:
Formatting zou in de frontend moeten gebeuren, ik heb liever niet hier al 'data loss'

Reply:
In de frontend zou je `Intl.NumberFormat` kunnen gebruiken, dan heb je qua localisation direct al je separators (comma vs. punt) goed staan https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Reply:
Helemaal eens. Maar altijd nog beter dan inches en miles 😆